src/inverseRobot.js

```diff
@@ -23,8 +23,22 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+
+  const inversedRobot = {};
+
+  for (let i = 0; i <= keys.length; i++) {
```

Comment:
better use for of with keys
or:
```suggestion
  for (const [key, value] of Object.entries(robot)) {
```